scipy/stats/_stats_py.py

```diff
@@ -8953,36 +8354,33 @@ def friedmanchisquare(*samples):
     References
     ----------
     .. [1] https://en.wikipedia.org/wiki/Friedman_test
-    .. [2] P. Sprent and N.C. Smeeton, "Applied Nonparametric Statistical
-           Methods, Third Edition". Chapter 6, Section 6.3.2.
+    .. [2] Demsar, J. (2006). Statistical comparisons of classifiers over
+           multiple data sets. Journal of Machine Learning Research, 7, 1-30.
 
     Examples
     --------
-    In [2]_, the pulse rate (per minute) of a group of seven students was
-    measured before exercise, immediately after exercise and 5 minutes
-    after exercise. Is there evidence to suggest that the pulse rates on
-    these three occasions are similar?
-
-    We begin by formulating a null hypothesis :math:`H_0`:
-
-        The pulse rates are identical on these three occasions.
 
-    Let's assess the plausibility of this hypothesis with a Friedman test.
+    In [2]_, the Friedman test is used to determine if four different
+    classifiers perform differently on 14 different data sets. The data is
+    given below, where each row corresponds to a different data set and each
+    column corresponds to a different classifier.
 
+    >>> import numpy as np
+    >>> x = np.array([[0.763, 0.599, 0.954, 0.628, 0.882, 0.936, 0.661, 0.583,\
+                       0.775, 1.0, 0.94, 0.619, 0.972, 0.957],
+                      [0.768, 0.591, 0.971, 0.661, 0.888, 0.931, 0.668, 0.583,\
+                       0.838, 1.0, 0.962, 0.666, 0.981, 0.978],
+                      [0.771, 0.590, 0.968, 0.654, 0.886, 0.916, 0.609, 0.563,\
+                       0.866, 1.0, 0.965, 0.614, 0.9751, 0.946],
+                      [0.798, 0.569, 0.967, 0.657, 0.898, 0.931, 0.685, 0.625,\
+                       0.875, 1.0, 0.962, 0.669, 0.975, 0.970]])
```

Comment:
Yes, looks like:
```suggestion
    >>> x = np.array([[0.763, 0.599, 0.954, 0.628, 0.882, 0.936, 0.661, 0.583,\
    ...                0.775, 1.0, 0.94, 0.619, 0.972, 0.957],
    ...               [0.768, 0.591, 0.971, 0.661, 0.888, 0.931, 0.668, 0.583,\
    ...                0.838, 1.0, 0.962, 0.666, 0.981, 0.978],
    ...               [0.771, 0.590, 0.968, 0.654, 0.886, 0.916, 0.609, 0.563,\
    ...                0.866, 1.0, 0.965, 0.614, 0.9751, 0.946],
    ...               [0.798, 0.569, 0.967, 0.657, 0.898, 0.931, 0.685, 0.625,\
    ...                0.875, 1.0, 0.962, 0.669, 0.975, 0.970]])
```

Reply:
🤦🏻 that's me not writing normal docstrings anymore

Reply:
Ok I figured out how to run smoke-docs locally and I think this fixed it, although I'm not sure how to make it look nicer/mode legible.

Reply:
I don't think there's anything important about the particular numbers, if this is just the docstring example.

Reply:
Ok, I guess the conclusion would be the same if I had less than 14 datasets, following the note

```
Due to the assumption that the test statistic has a chi squared
distribution, the p-value is only reliable for n > 10 and more than
6 repeated samples.
```